src/components/attachment-dialog.js

```diff
@@ -107,7 +106,7 @@ class AttachmentDialog extends BaseMixin(LitElement) {
 		if (this.attachment) {
 			const { name, filepath } = this.attachment;
 			this.attachmentUrl = window.navigator.msSaveOrOpenBlob ?
-				window.navigator.msSaveOrOpenBlob(name, filepath) : window.URL.createObjectURL(this.attachment);
+				window.navigator.msSaveOrOpenBlob(name, name) : window.URL.createObjectURL(this.attachment);
```

Comment:
It's kind of weird, but it should be:
```suggestion
				window.navigator.msSaveOrOpenBlob(blob, name) : window.URL.createObjectURL(this.attachment);
```
see https://developer.mozilla.org/en-US/docs/Web/API/Navigator/msSaveOrOpenBlob